x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -1325,6 +1325,9 @@ private static void withBaseUpdateApiKeyFields(final XContentBuilder builder, fi
                 // because it replaces any metadata previously associated with the API key
                 builder.field("metadata", baseUpdateApiKeyRequest.getMetadata());
             }
+            if (baseUpdateApiKeyRequest.getExpiration() != null) {
+                builder.field("expiration", baseUpdateApiKeyRequest.getExpiration());
```

Comment:
Not a strong preference, but I'd stay consistent with the create API key event log, that is explicitly log `expiration` as `null`, i.e. sth like: `field("expiration", expiration != null ? expiration.toString() : null)`

Reply:
Good point! Changed it.